hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieWriterUtils.scala

```diff
@@ -153,6 +152,31 @@ object HoodieWriterUtils {
     }
   }
 
+  private val expressionPayloadClassName = "org.apache.spark.sql.hudi.command.payload.ExpressionPayload"
+  private val validateDuplicateKeyPayloadClassName = "org.apache.spark.sql.hudi.command.ValidateDuplicateKeyPayload"
```

Comment:
Rename them to `EXPRESSION_PAYLOAD_CLASS_NAME`, etc.?

Reply:
it will fail the style

Reply:
`DataSourceReadOptions.scala` has such naming convention.  Should it be `val` instead of `private val`?

Reply:
Fixed.